[2025-07-09T06:22:32.423+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-09T06:22:32.440+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_airflare_data.transform manual__2025-07-09T06:21:25.645739+00:00 map_index=1 [queued]>
[2025-07-09T06:22:32.446+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_airflare_data.transform manual__2025-07-09T06:21:25.645739+00:00 map_index=1 [queued]>
[2025-07-09T06:22:32.446+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-07-09T06:22:32.456+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): transform> on 2025-07-09 06:21:25.645739+00:00
[2025-07-09T06:22:32.468+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5330) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-09T06:22:32.470+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_airflare_data', 'transform', 'manual__2025-07-09T06:21:25.645739+00:00', '--job-id', '460', '--raw', '--subdir', 'DAGS_FOLDER/Untitled-1.py', '--cfg-path', '/tmp/tmpmirnh69l', '--map-index', '1']
[2025-07-09T06:22:32.471+0000] {standard_task_runner.py:63} INFO - Started process 5340 to run task
[2025-07-09T06:22:32.471+0000] {standard_task_runner.py:91} INFO - Job 460: Subtask transform
[2025-07-09T06:22:32.534+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_airflare_data.transform manual__2025-07-09T06:21:25.645739+00:00 map_index=1 [running]> on host e6a32b72ede0
[2025-07-09T06:22:32.640+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_airflare_data' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T06:21:25.645739+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T06:21:25.645739+00:00'
[2025-07-09T06:22:32.642+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-09T06:22:32.642+0000] {Untitled-1.py:136} INFO - Данные приняты из Xcom, начат процесс очистки
[2025-07-09T06:22:32.643+0000] {Untitled-1.py:58} INFO - Создание подключения к S3
[2025-07-09T06:22:32.643+0000] {Untitled-1.py:62} INFO - Подключение установлено, загрузка данных из S3
[2025-07-09T06:22:32.655+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T06:22:32.656+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T06:22:32.789+0000] {Untitled-1.py:68} INFO - Данные выгружены из S3
[2025-07-09T06:22:32.794+0000] {Untitled-1.py:145} INFO - Преобразование завершено, начинается передача DF в S3
[2025-07-09T06:22:32.795+0000] {Untitled-1.py:26} INFO - Создание подключения к S3
[2025-07-09T06:22:32.795+0000] {Untitled-1.py:31} INFO - Подключение установлено
[2025-07-09T06:22:32.802+0000] {base.py:84} INFO - Using connection ID 'minio' for task execution.
[2025-07-09T06:22:32.804+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='minio', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T06:22:33.155+0000] {Untitled-1.py:37} INFO - Загрузка данных в S3
[2025-07-09T06:22:33.439+0000] {Untitled-1.py:48} INFO - Загрузка завершена успешно
[2025-07-09T06:22:33.439+0000] {Untitled-1.py:153} INFO - Данные успешно сохранены в S3, передаю их в Xcom
[2025-07-09T06:22:33.440+0000] {Untitled-1.py:157} INFO - данные успешно очищены и записаны в хранилище
[2025-07-09T06:22:33.440+0000] {python.py:237} INFO - Done. Returned value was: {'bucket': 'airflare', 'key': 'temp/df_HKT_IKT.json'}
[2025-07-09T06:22:33.441+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-09T06:22:33.461+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_airflare_data, task_id=transform, run_id=manual__2025-07-09T06:21:25.645739+00:00, map_index=1, execution_date=20250709T062125, start_date=20250709T062232, end_date=20250709T062233
[2025-07-09T06:22:33.488+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-09T06:22:33.511+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T06:22:33.513+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
